src/lib/storage/chunk_encoder.hpp

```diff
@@ -47,17 +52,39 @@ class ChunkEncoder {
   static void encode_chunk(const std::shared_ptr<Chunk>& chunk, const std::vector<DataType>& data_types,
                            const ChunkEncodingSpec& encoding_spec);
 
+  /**
+   * @brief Encodes a chunk using the same column-encoding spec
```

Comment:
the same as what?

Reply:
Is “… using a single column-encoding spec” better?

Reply:
yes